Makefile

```diff
@@ -165,13 +165,13 @@ endef
 $(eval $(call create_help_target,arkouda-help,ARKOUDA_HELP_TEXT))
 
 # Set the arkouda server version from the VERSION file
-VERSIONFILE=./VERSION
+VERSION=$(shell python -m arkouda|tail -n 1)
```

Comment:
I see you added a `__main__.py` that prints the version on module load to ensure this Makefile line works, but I'm wondering if it would be more robust to do something like `VERSION=$(shell python -c "import arkouda; print(arkouda.__version__)")`?

Reply:
RE: `versioneer.py` - The versioneer docs say this is necessary if you want `setup.py` to be able to compute the version (see the section [Theory of Operation](https://github.com/python-versioneer/python-versioneer#theory-of-operation) for details).

RE: the shell operation and `main.py`, Yes! I can take that out and streamline it a bit.

There are 2 basic options:
1. Directly from versioneer
```
VERSION=$(shell python -c "import versioneer; print(versioneer.get_versions()[\"version\"])")
```

2. via import; note: Pyfiglet prints out the ascii-art any time arkouda is imported so I still need `tail` to filter down to the last line, but here's that version:
```
VERSION=$(shell python -c "import arkouda; print(arkouda.__version__)"|tail -n 1)
```

I'll push a fix/change momentarily

Reply:
Ok, that makes sense.

Makefile change looks good!